Providers/Scripts/2.4x-2.5x/Scripts/nxOMSGenerateInventoryMof.py

```diff
@@ -148,7 +154,22 @@ def GenerateInventoyConfContents(FeatureName, Instances, RunIntervalInSeconds, T
   # Force upload even if the data has not changed \n \
   force_send_run_interval 24h \n \
   log_level warn \n \
-</filter> \n'
+</filter> \n
+<match ' + Tag + '> \n \
+  type out_oms_' + Tag + '\n \
+  log_level trace \n \
+  num_threads 5 \n \
+  buffer_chunk_limit 5m \n \
+  buffer_type file \n \
+  buffer_path /var/opt/microsoft/omsagent/state/out_oms_' + Tag + '*.buffer \n \
+  buffer_queue_limit 10 \n \
+  buffer_queue_full_action drop_oldest_chunk \n \
+  flush_interval 20s \n \
+  retry_limit 10 \n \
+  retry_wait 30s \n \
+  max_retry_wait 9m \n \
+' + configurations + '\
+</match> \n'
```

Comment:
I'm not sure this multi-line string is formatted correctly; GitHub is highlighting the ends in red... I believe in Ruby you still have to give a quote and backslash at the end of each line to communicate continuation ([reference](http://stackoverflow.com/questions/2337510/ruby-can-i-write-multi-line-string-with-no-concatenation)).